hail/python/hail/fs/google_fs.py

```diff
@@ -139,3 +139,6 @@ def rmtree(self, path: str):
         if self._is_local(path):
             rmtree(path)
         self.client.rm(path, recursive=True)
+
+    def supports_scheme(self, scheme: str) -> bool:
+        return scheme in ("gs", "")
```

Comment:
Should also support `file` as a scheme.

Reply:
For URLs that don't start with `gs://`, GoogleCloudStorageFS uses `os` methods and `open`, which don't support `file://` URLs.